test/measures/measures-schema-spec.js

```diff
@@ -7,22 +7,25 @@ const testFilesFolder = 'test/measures/examples/quality/';
 const qualityMeasureExampleFilenames = fs.readdirSync(testFilesFolder);
 
 describe('measures schema validates json', function() {
-  describe('measures-data.json', function() {
-    let capturedStdout = null;
+  const performanceYears = [2017, 2018];
+  for (const year of performanceYears) {
+    describe(year + ' measures-data.json', function() {
```

Comment:
This is a poor diff... everything below this line through the end of the file is unchanged other than being indented by two spaces since it's now inside a for..of loop.